### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by maoschanz**
i'm not 100% it works because meson needs packages which don't exist

### CODE REVIEW

File: AudioCutter/application.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def _on_night_mode(self, action, *args):
 
     def _on_shortcuts(self, *args):
         """Shows keyboard shortcuts."""
-        shortcuts = ShortuctsWindow()
+        shortcuts = ShortcutsWindow()
```
Comment (bilelmoussaoui) [inline AudioCutter/application.py:122]: Bien vu!

File: po/POTFILES [hunk 2]
```diff
@@ -0,0 +1,13 @@
+AudioCutter/application.py
+AudioCutter/const.py
+AudioCutter/widgets/about.py
+AudioCutter/widgets/actionbar.py
```
Comment (bilelmoussaoui) [inline po/POTFILES:4]: Peut-être rajouter l'appdata & le desktop/gschema aussi?

**Comment by bilelmoussaoui** [comment]
Which packages exactly?

**Comment by maoschanz** [comment]
meson complained about gobject-introspection, and installing things with related names didn't soothe its wrath so i gave up

**Comment by bilelmoussaoui** [comment]
aha, it's a WIP project, will merge and try it later ^^
Thanks!

File: po/POTFILES (hunk 2 above)
Comment (maoschanz) [reply po/POTFILES:4]: oh, that is why it was the app id?

Comment (bilelmoussaoui) [reply po/POTFILES:4]: not really, I just used the app-id because why not

**Pull request merged**